apps/src/javalab/javalabRedux.js

```diff
@@ -40,6 +43,12 @@ export const setSource = (filename, source, isVisible = true) => ({
   isVisible
 });
 
+export const setSourceVisibility = (filename, isVisible) => ({
+  type: SET_SOURCE_VISIBILITY,
```

Comment:
NIT (and up to you if you want to fix): We haven't been following this convention so far in this file, but actions should generally be named as events rather than setters. [Details from the Redux Style Guide](https://redux.js.org/style-guide/style-guide#model-actions-as-events-not-setters). SOURCE_VISIBILITY_UPDATED or similar would indicate more clearly what's happening here i.e. an event happened and therefore redux should update its state.